package-lock.json

```diff
@@ -38,8 +38,8 @@
     },
```

Comment:
Question: What's changed here in order for us to generate a new lock?

Reply:
I think i generated the previous one with an old version of node